### PR TITLE
Pedestals on S3DF

### PR DESCRIPTION
**Pull request by vespos**
## Description
Make makepeds compatible with running on the S3DF, without changing the well established workflow

## How Has This Been Tested?
Tested locally S3DF for data on the FFB.
Need more test for data on S3DF, from hutch opr accounts and with data in transfer

## Where Has This Been Documented?
Docstrings in scripts

### CODE REVIEW

File: scripts/makepeds [hunk 1]
```diff
@@ -222,7 +222,7 @@ else
     elif [[ $FFB == 2 ]]; then
         echo calling on SDF system: makepeds_psana $@
 	#we need to change paths here as directories are completely independent
-	DIR=`echo $DIR | sed 's/\/cds\/group\/pcds/sdf\/group\/lcls\/tools/g' | sed 's/\/reg\/g\/pcds/\/sdf\/group\/lcls\/ds\/tools/g'`
+	DIR=`echo $DIR | sed 's/\/cds\/group\/pcds/\/sdf\/group\/lcls\/tools/g' | sed 's/\/reg\/g\/pcds/\/sdf\/group\/lcls\/ds\/tools/g'`
```
Comment (silkenelson) [inline scripts/makepeds:223]: Does the 'cds' replace not also need the 'ds' at lcls/ds/tools?

Comment (silkenelson) [reply scripts/makepeds:223]: Also, lines 205-208 can go as they are doing what lines 2245 et al now do better. I could not figure out how to comment there.

**Review by silkenelson — COMMENTED** [review]
Minor comments.

File: scripts/makepeds (hunk 1 above)
Comment (vespos) [reply scripts/makepeds:223]: That's right!
ANd I will check the lines you mention. You mean line 245 though, right?

**Review by silkenelson — APPROVED** [review]
I think we only need 1 cpu-per-task, but I had not finished testing. Either way, given the total number of cores, they should be able to fit into a single node when run non-exclusively, so this is a step forward.

**Pull request merged**